### PR TITLE
Add missing docs

### PR DESCRIPTION
**Pull request by OsvaldoRosado**
Addresses #214

### CODE REVIEW

File: Library/Client.ts [hunk 1]
```diff
@@ -145,14 +145,44 @@ class Client {
...
+     * Use this when you need to perform custom logic to record the duration of the request.
+     * 
+     * This call will also add outgoing headers to the supplied response object for correlating telemetry across different services.
+     * @param request              http.ServerRequest - the request object to monitor
+     * @param response             http.ServerResponse - the response object to monitor
+     * @param elapsedMilliseconds  number - the elapsed time taken to handle this request in milliseconds
+     * @param properties           map[string, string] - additional data used for filtering in the portal. Defaults to empty.
+     * @param error                any - an object indicating the request was unsuccessful. This object will be recorded with the request telemetry.
+     */
     public trackRequestSync(request: http.ServerRequest, response: http.ServerResponse, ellapsedMilliseconds?: number, properties?: { [key: string]: string; }, error?: any) {
```
Comment (SergeyKanzhelev) [inline Library/Client.ts]: Should we have `startRequest` `stopRequest` instead? `Sync` is really misleading in this context.

Comment (OsvaldoRosado) [inline Library/Client.ts]: I agree, this name is confusing. I'm not sure what start/stop would gain over the non-"sync" version of this call however

Comment (KamilSzostak) [inline Library/Client.ts]: I had the same comment when the `*sync` methods were added - #75. According to the author, this follows the Node.js naming convention, but I'm open to renaming them in the future.

Comment (SteveALee) [inline Library/Client.ts]: Agree, To me the sync implied it waits for the response, the exact opposite of actual behaviour. But once you grok it it's OK

**Pull request merged**